provisioning/miniprov/hare_mp/main.py

```diff
@@ -237,7 +237,7 @@ def shutdown_cluster():
 def list2dict(nodes_data_hctl: list) -> dict:
     node_info_dict = {}
     for node in nodes_data_hctl:
-        node_svc_info = {}
+        node_svc_info: dict = {}
```

Comment:
Dict[key_type, value_type] instead of dict please. More info: https://stackoverflow.com/questions/37087457/difference-between-defining-typing-dict-and-dict

Reply:
Done